src/MetricHandler.js

```diff
@@ -103,6 +106,20 @@ export class MetricHandler {
         if (splitted[0] === requestedMetrics[i]) {
           matchingAggregatesObj[splitted[1]] = true
           matchingAggregatesCount += 1
+          if (splitted[1] === 'count' || splitted[1] === 'raw') {
+            let pointsRaw = 0
+            if (splitted[1] === 'count') {
+              for (const entry in myData[curMetricName].data) {
+                pointsRaw += myData[curMetricName].data[entry].value
+              }
+              this.store.dispatch('metrics/updateDataPoints', { metricKey: splitted[0], pointsAgg: myData[curMetricName].data.length, pointsRaw: pointsRaw })
+              pointCountAgg += myData[curMetricName].data.length
+            } else {
+              pointsRaw = myData[curMetricName].data.length
+              this.store.dispatch('metrics/updateDataPoints', { metricKey: splitted[0], pointsAgg: 0, pointsRaw: pointsRaw })
```

Comment:
`pointsAgg: 0` that is not the best default here. `0` is a valid value for the number of aggregates. I think it should be `null` instead.